### PR TITLE
Dexsearch: add ubers alias

### PR DESCRIPTION
**Pull request by CheeseMuffin**

### CODE REVIEW

**Comment by jumbowhales** [comment]
should you do this for lc ubers too?

File: chat-plugins/datasearch.js [hunk 1]
```diff
@@ -348,7 +348,7 @@ if (process.send && module === process.mainModule) {
 
 function runDexsearch(target, cmd, canAll, message) {
 	let searches = [];
-	let allTiers = {'uber':'Uber', 'ou':'OU', 'bl':"BL", 'uu':'UU', 'bl2':"BL2", 'ru':'RU', 'bl3':"BL3", 'nu':'NU', 'bl4':"BL4", 'pu':'PU', 'nfe':'NFE', 'lc uber':"LC Uber", 'lc':'LC', 'cap':"CAP", 'caplc': "CAP LC", 'capnfe': "CAP NFE"};
+	let allTiers = {'uber':'Uber', 'ubers': 'Uber', 'ou':'OU', 'bl':"BL", 'uu':'UU', 'bl2':"BL2", 'ru':'RU', 'bl3':"BL3", 'nu':'NU', 'bl4':"BL4", 'pu':'PU', 'nfe':'NFE', 'lcuber':"LC Uber", 'lcubers': "LC Uber", 'lc':'LC', 'cap':"CAP", 'caplc': "CAP LC", 'capnfe': "CAP NFE"};
```
Comment (panpawn) [inline chat-plugins/datasearch.js]: This should really have consistent spacing between all elements of ``allTiers``

**Comment by CheeseMuffin** [comment]
Yeah may as well, also made it more lenient with spacing (so `/ds lcuber` and `/ds lc ubers' both work now)

File: chat-plugins/datasearch.js [hunk 2]
```diff
@@ -348,7 +348,7 @@ if (process.send && module === process.mainModule) {
 
 function runDexsearch(target, cmd, canAll, message) {
 	let searches = [];
-	let allTiers = {'uber':'Uber', 'ou':'OU', 'bl':"BL", 'uu':'UU', 'bl2':"BL2", 'ru':'RU', 'bl3':"BL3", 'nu':'NU', 'bl4':"BL4", 'pu':'PU', 'nfe':'NFE', 'lc uber':"LC Uber", 'lc':'LC', 'cap':"CAP", 'caplc': "CAP LC", 'capnfe': "CAP NFE"};
+	let allTiers = {'uber':'Uber', 'ubers':'Uber', 'ou':'OU', 'bl':"BL", 'uu':'UU', 'bl2':"BL2", 'ru':'RU', 'bl3':"BL3", 'nu':'NU', 'bl4':"BL4", 'pu':'PU', 'nfe':'NFE', 'lcuber':"LC Uber", 'lcubers':"LC Uber", 'lc':'LC', 'cap':"CAP", 'caplc': "CAP LC", 'capnfe': "CAP NFE"};
```
Comment (panpawn) [inline chat-plugins/datasearch.js]: ``'bl4':"BL4", 'pu':'PU', 'nfe':'NFE', 'lcuber':"LC Uber", 'lcubers':"LC Uber",``

Either use all tic marks or all quotes; it should be consistent

Comment (CheeseMuffin) [inline chat-plugins/datasearch.js]: It was already " for multi word tiers and ' for single word tiers so I kept it consistent with that

Comment (panpawn) [inline chat-plugins/datasearch.js]: I don't think that should really matter. Also,
>'bl4':"BL4"

File: chat-plugins/datasearch.js [hunk 3]
```diff
@@ -348,7 +348,7 @@ if (process.send && module === process.mainModule) {
 
 function runDexsearch(target, cmd, canAll, message) {
 	let searches = [];
-	let allTiers = {'uber':'Uber', 'ou':'OU', 'bl':"BL", 'uu':'UU', 'bl2':"BL2", 'ru':'RU', 'bl3':"BL3", 'nu':'NU', 'bl4':"BL4", 'pu':'PU', 'nfe':'NFE', 'lc uber':"LC Uber", 'lc':'LC', 'cap':"CAP", 'caplc': "CAP LC", 'capnfe': "CAP NFE"};
+	let allTiers = {'uber':'Uber', 'ubers':'Uber', 'ou':'OU', 'bl':'BL', 'uu':'UU', 'bl2':'BL2', 'ru':'RU', 'bl3':'BL3', 'nu':'NU', 'bl4':'BL4', 'pu':'PU', 'nfe':'NFE', 'lcuber':"LC Uber", 'lcubers':'LC Uber', 'lc':'LC', 'cap':'CAP', 'caplc': 'CAP LC', 'capnfe': 'CAP NFE'};
```
Comment (panpawn) [inline chat-plugins/datasearch.js]: this still has inconsistent spacing :|

compare ``'uber':'Uber',`` and ``'caplc': 'CAP LC', 'capnfe': 'CAP NFE'`` for example

**Comment by CheeseMuffin** [comment]
Bump any other problems with this?

**Comment by Zarel** [comment]
This is good; thanks!

**Pull request merged**